variables.tf

```diff
@@ -82,6 +82,7 @@ variable "okta" {
   type = object({
     enabled         = optional(bool, false)
     aws_secret_name = optional(string, "")
+    scope           = optional(string, "openid")
```

Comment:
@rbapst-tamedia [minor] It would be good to provide an example of multiple scopes to be set in the variable. OIDC expects them joined via a whitespace (e.g. `openid groups`).

An alternative would be to set the input as a list and then we do a `.join(" ")`